### ISSUE TITLE
Azure Storage Explorer fails to enlist files in a blob container pre-existing folder by issuing "The first argument must be of type string or an instance of Buffer, ArrayBuffer, or Array or an Array-like Object. Received undefined" error message

### ISSUE THREAD

**Issue by Arash-Sabet** [microsoft/AzureStorageExplorer#6035]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.25.1

### Regression From

1.23.x

### Architecture

x64

### Storage Explorer Build Number

20220809.7

### Platform

Windows

### OS Version

Windows 11

### Bug Description

One won't be able to view the list of files stored in a blob container folder. Files can be viewed if they are not uploaded to a blob container folder.

### Steps to Reproduce

1. Launch Storage Explorer
2. Choose a pre-existing folder
3. Upload a file to that folder
4. You won't see the uploaded file

This issue does not occur when a new folder is created.

### Actual Experience

![image](https://user-images.githubusercontent.com/26050123/185108716-292f5d4f-6ac3-42c7-8101-888e73f92329.png)

### Expected Experience

We expect to see all files in a pre-existing folder.

**Comment by MRayermannMSFT** [comment]
@Arash-Sabet what version of Azurite are you using?

**Comment by Arash-Sabet** [comment]
@MRayermannMSFT The azurite instance exists in the docker and it has to be the latest version per the following yaml entry:

```
image: "mcr.microsoft.com/azure-storage/azurite:latest"
```

Issuing `azurite --version` through the docker's console reveals version 3.16.0

**Comment by JasonYeMSFT** [comment]
I believe this is a duplicate issue of https://github.com/microsoft/AzureStorageExplorer/issues/6008. You can workaround it by manually setting the content MD5 on the problematic blob.

**Comment by Arash-Sabet** [comment]
> I believe this is a duplicate issue of #6008. You can workaround it by manually setting the content MD5 on the problematic blob.

The blob is not problematic and the issue is that a pre-existing folder in a blob container cannot be viewed to browse its contents. I don't if your solution is right.

**Comment by JasonYeMSFT** [comment]
@Arash-Sabet Could you write a script to list the blobs in the affected folder and check if all the blobs in it have their Content-MD5 set? This is a common issue that can be caused by writing with scripts because the script author sometimes forgets to set the Content-Md5 after committing the content of the blob.

**Comment by Arash-Sabet** [comment]
@JasonYeMSFT I'm afraid that I won't be able to make time to write a script. 

Just as a background, this is how the folder created in the blob container in our docker project:

- We defined the environment variable of `WEBSITE_SITE_NAME=myfolder` in the `.env` file
- We build our azure function image like the following pattern *(I removed unnecessary environment variables)*:

```yaml
myfunctions:
    image: ${DOCKER_REGISTRY-}myfunctionapp
    container_name: myfunction
    build: 
      context: .
      dockerfile: MyFunctionApp/Dockerfile
    environment: 
      - AzureWebJobsStorage
      - FUNCTIONS_WORKER_RUNTIME
      - WEBSITE_SITE_NAME
    ports: 
      - 18686:80    
    depends_on: 
      - storageemulator
```

When the docker runs, I think it will have the azure function app create the folder automatically. There is no explicit instruction in the docker yaml file to create the folder.
